.mailmap

```diff
@@ -1,20 +1,62 @@
+Akshay Subramanian <akshay.sub12@gmail.com>
 Alli Black <black.alli@gmail.com>
+Andreas Tille <tille@debian.org>
+Anna Parker <50943381+anna-parker@users.noreply.github.com>
+Anna Parker <anya.m.parker@gmail.com>
 Barney Potter <barney.potter.24@gmail.com>
+Benjamin <benjamin.otter@stud.unibas.ch>
+Benjamin Otter <benjamin.otter@stud.unibas.ch>
+Cameron Devine <camerondtrain@gmail.com>
+Carlosbg <bogomcar@gmail.com>
+Carol Willing <carolcode@willingconsulting.com>
+Chris Woszczak <cwoszczak@protonmail.com>
+Christian Clauss <cclauss@me.com>
+Christopher Tomkins-Tinch <tomkinsc@broadinstitute.org>
+Cornelius Roemer <cornelius.roemer@gmail.com>
+Daniel Dye <dye.daniel@gmail.com>
+Danielle Kain <dkain@chanzuckerberg.com>
+Dependabot <49699333+dependabot[bot]@users.noreply.github.com>
+Eddie Lebow <elebow@users.noreply.github.com>
+Elias Harkins <eli.harkins@gmail.com>
 Emma Hodcroft <emmahodcroft@gmail.com>
 Emma Hodcroft <freezefire666@yahoo.com>
+Eric Danielson <edanielson@twitter.com>
+Étienne Mollier <63921018+emollier@users.noreply.github.com>
+Felix Yan <felixonmars@archlinux.org>
 Gytis Dudas <gytisdudas@gmail.com>
+James Hadfield <hadfield.james@gmail.com>
 James Hadfield <jameshadfield@users.noreply.github.com>
 James Hadfield <jh22@sanger.ac.uk>
+Jennifer Chang <jenchang@iastate.edu>
+Jennifer Chang <jennifer.chang.bioinform@gmail.com>
 John Huddleston <huddlej@gmail.com>
+John SJ Anderson <janders4@fredhutch.org>
+Jover Lee <joverlee521@gmail.com>
+Julien Bordellier <git@julienbordellier.com>
+Kairsten Fay <kairsten.fay@gmail.com>
 Louise Moncla <lhmoncla@gmail.com>
+Mathias Walter <mathias@walter.gd>
+Mingye Wang <arthur200126@gmail.com>
+Misja Ilcisin <milcisin@fredhutch.org>
+Nextstrain bot <hello@nextstrain.org>
+Pierre Barrat <p.barrat@live.fr>
+Rhys Kidd <rhyskidd@gmail.com>
 Richard Neher <neher@login10.cluster.bc2.ch>
 Richard Neher <neher@login11.cluster.bc2.ch>
 Richard Neher <neher@login20.cluster.bc2.ch>
 Richard Neher <richard.neher@tuebingen.mpg.de>
 Richard Neher <richard.neher@unibas.ch>
+root <root@MATZENOTE.localdomain>
```

Comment:
We should map this to someone meaningful. It looks like this came from a commit in #467, so

```
Mathias Walter <mathias@walter.gd> root <root@MATZENOTE.localdomain>
```